llvm/lib/Analysis/DXILResource.cpp

```diff
@@ -879,8 +883,122 @@ SmallVector<dxil::ResourceInfo *> DXILResourceMap::findByUse(const Value *Key) {
 
 //===----------------------------------------------------------------------===//
 
+void DXILResourceBindingInfo::populate(Module &M, DXILResourceTypeMap &DRTM) {
+  struct Binding {
+    ResourceClass ResClass;
+    uint32_t Space;
+    uint32_t LowerBound;
+    uint32_t UpperBound;
+    Binding(ResourceClass RC, uint32_t Sp, uint32_t LB, uint32_t UB)
+        : ResClass(RC), Space(Sp), LowerBound(LB), UpperBound(UB) {}
```

Comment:
Better to just shadow the member names rather than make the parameters unreadable.
```suggestion
    Binding(ResourceClass RC, uint32_t Space, uint32_t LowerBound, uint32_t UpperBound)
        : ResClass(RC), Space(Space), LowerBound(LowerBound), UpperBound(UpperBound) {}
```